lib/Optimize.cpp

```diff
@@ -204,7 +204,7 @@ void OptimizeModule(const EntityLifter &lifter, llvm::Module &module,
   fpm.addPass(llvm::VerifierPass());
   fpm.addPass(llvm::DSEPass());
   fpm.addPass(llvm::VerifierPass());
-  fpm.addPass(llvm::SROAPass());
+  fpm.addPass(llvm::SROAPass(llvm::SROAOptions::ModifyCFG));
```

Comment:
This pass now takes in some configuration (the other option is called `PreserveCFG` and restricts what optimizations are allowed).